### PR TITLE
Mordi490.GitHub.io 15 bugfix sha 4862079a7ef81c7e374138535099a06569b3c546

### PR DESCRIPTION
**Pull request by 0us**
this builds on pr https://github.com/Mordi490/mordi490.github.io/pull/9, so please merge that one first

fix bug

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,10 +12,21 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inconsolata&display=swap"
+      rel="stylesheet"
+    />
     <h2>Greetings retard</h2>
   </head>
 
   <body>
+    <script src="js/bugfix.js"></script>
```
Comment (Mordi490) [inline index.html:29]: haha

Comment (0us) [reply index.html:29]: ? please take this more seriously

Best Regards
Team Natron

Comment (markusranda) [reply index.html:29]: Why did you not use correct indentation @0us  ? ? ?

**Comment by markusranda** [comment]
Hello, where are the improvements ? ? Is this site totally abandoned?

**Comment by 0us** [comment]
Have you no regard for the future of this site? Please see to these improvements immediately.

**Pull request closed without merging**